definitions/puma_config.rb

```diff
@@ -151,17 +151,16 @@
     end
   end
 
-
-  logrotate_app puma_params[:name] do
-    cookbook "logrotate"
-    path [ puma_params[:stdout_redirect], puma_params[:stderr_redirect] ]
-    frequency "daily"
-    rotate 30
-    size "5M"
-    options ["missingok", "compress", "delaycompress", "notifempty", "dateext"]
-    variables puma_params
-    only_if params[:logrotate]
+  if params[:logrotate]
```

Comment:
i can probably remove... the real reason is that I didn't want all the logrotate deps